app/javascript/components/Discover/Nav.tsx

```diff
@@ -48,7 +55,7 @@ const generateTaxonomyItemsForMenu = (wholeTaxonomy: Taxonomy[]) => {
       slugs.unshift(curr.slug);
       curr = curr.parent_key ? taxonomyMap.get(curr.parent_key) : undefined;
     }
-    return `/${slugs.join("/")}`;
+    return `${discoverUrl}${slugs.join("/")}`;
```

Comment:
@lukicenturi thanks for the contribution!

The code expects this to be a relative path in other places: 
![image](https://github.com/user-attachments/assets/839c81ff-5f8c-4e1f-98a7-4b803a439fda)

Would you be able to check it?

Reply:
Ups, I will take a look

Reply:
@ershad Thanks for the review, I push a fix, kindly check again